wp-content/themes/workday/partials/content-single.php

```diff
@@ -0,0 +1,62 @@
+<?php
+/**
+ * The template for displaying content in the single.php template
+ */
+?>
+
+<article id="post-<?php the_ID(); ?>" <?php post_class( 'hnews item' ); ?> itemscope itemtype="https://schema.org/Article">
+
+    <?php do_action('largo_before_post_header'); ?>
+    
+    <?php 
+    
+    add_filter( 'body_class', function( $classes ) {
+        $classes[] = 'photo-header';
+        return $classes;
+    } );
+    
+    workday_photo_header_tag( get_post_thumbnail_id() ); 
+
+    ?>
+        <div class="featured-image-bg-layer">
+        </div>
+        <div class="featured-image-container-content">
+            <?php
+
+            if( get_post_thumbnail_id() ) {
+                echo '<div class="featured-image-wrapper"><img class="featured-image-container-mobile-image" src="'.wp_get_attachment_url( get_post_thumbnail_id() ).'"></div>';
+            }
+
+            ?>
+            <?php largo_maybe_top_term(); ?>
+            <h1 class="entry-title" itemprop="headline"><?php the_title(); ?></h1>
+            <h5 class="byline"><?php largo_byline( true, true, get_the_ID() ); ?></h5>
+
+            <?php if ( ! of_get_option( 'single_social_icons' ) == false ) { ?>
+                <ul class="photo-header-social-links">
+                    <?php largo_social_links(); ?>
```

Comment:
Instead, should we use the largo_post_social_links function to output these links with sharing intent? https://github.com/INN/largo/blob/v0.6.4/inc/post-social.php#L26

Reply:
Probably. I wasn't aware of that one. I'll swap them out.

Reply:
@benlk There's no way to filter the output list of `largo_post_social_links` so we can redo the order and also display LinkedIn and Pinterest, is there? We'll probably need to rebuild that function to suit the icons that are requested here.

Reply:
No, there isn't. (Unless you feel like parsing HTML.) It's touched upon in https://github.com/INN/largo/issues/1708, for an undetermined future Largo version.

I think in other themes we've straight-up redefined `largo_post_social_links()` (it is a pluggable function) in order to add new social buttons or reorder them.

Reply:
@MirandaEcho Before I go too far down the rabbit hole, is it ok if I do this?

Basically, we have 2 functions, one that displays links to the clients social media (which is the function I was using) and one that displays links *to share* the post on social media.

If we want to use function #2 so users can share the post from the buttons we will need to create a custom function based off of it since it does not show the correct social media sites

Reply:
@joshdarby it's fine to rearrange the social icons. as far as I understand, that's all we would need to filter for with the function that displays links to share. is that true? or is there further filtering needed.

TL;DR:
We don't have to follow the mockup exactly for the social icon display, so let's figure out a quick/easy way to use the sharing function.

Reply:
@kaylima We would also need to add in functionality to share to Pinterest and LinkedIn.